### PR TITLE
Added a sleep before hammer ping so that foreman-task service cleanly comes up

### PR DESCRIPTION
**Pull request by sghai**
handles #46

### CODE REVIEW

**Comment by sthirugn** [comment]
As per Sachin, adding 3 minutes didnt solve the problem, so he added 5 minutes.

**Comment by sthirugn** [comment]
ACK

File: roles/sat6repro/tasks/main.yml [hunk 1]
```diff
@@ -91,11 +91,14 @@
...
-- name: Restart katello-service and test satellite
-  command: "{{ item }}"
-  with_items:
-    - katello-service start
-    - hammer ping
+- name: Restart katello-service
+  command: katello-service start
+
+- name: Wait for foreman-tasks service to comes up
+  shell: sleep 5m
```
Comment (sghai) [inline roles/sat6repro/tasks/main.yml]: @sthirugn : I was reading about ansible before making change and realized that pause is for playbook but not for tasks. But as you suggested 'pause' looks more promising. will update the PR.

**Comment by sthirugn** [comment]
Thanks @sghai

**Pull request merged**